packages/fx-core/src/common/m365/packageService.ts

```diff
@@ -74,6 +74,71 @@ export class PackageService {
     }
   }
 
+  @hooks([ErrorContextMW({ source: M365ErrorSource, component: M365ErrorComponent })])
+  public async sideLoadXmlManifest(token: string, manifestPath: string): Promise<[string, string]> {
+    try {
+      const data = await fs.readFile(manifestPath);
+      const content = new FormData();
+      content.append("package", data);
+      content.append("culture", "en-us");
```

Comment:
Does culture depend on the xml content, or this hard-coded en-us already enough?

Reply:
@millerds told me that culture "en-us" needs to be included in the form data. I'll double check with him.